### PR TITLE
Added TomTom layers

### PR DESCRIPTION
**Pull request by yecarrillo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,19 @@ In order to use ArcGIS maps, you must [register](https://developers.arcgis.com/e
...
+
+In order to use TomTom layers, you must [register](https://developer.tomtom.com/user/register). Once registered, you can create an `apikey` which you have to pass to `L.tileLayer.provider` in the options:
+
+```Javascript
+L.tileLayer.provider('TomTom', {
+    apikey: '<insert your API key here>'
+}).addTo(map);
+```
+
+[Available HERE layers](http://leaflet-extras.github.io/leaflet-providers/preview/#filter=HERE)
```
Comment (brunob) [inline README.md]: Should be a link related to tomtom layers instead of HERE ;)

File: preview/preview.js [hunk 2]
```diff
@@ -66,7 +66,9 @@
 			'OpenPtMap',
 			'OpenRailwayMap',
 			'OpenFireMap',
-			'SafeCast'
+			'SafeCast',
+			'TomTom.Hybrid',
+			'TomTom.Labels'
```
Comment (brunob) [inline preview/preview.js]: Are you sure that the API key provided will handle the amount of hits on demo page ?

File: README.md (hunk 1 above)
Comment (yecarrillo) [reply README.md]: Done!

File: preview/preview.js (hunk 2 above)
Comment (yecarrillo) [reply preview/preview.js]: Not sure. This key is from my personal Dev account with "2,500 TomTom Maps API transactions for free on a daily basis".

Comment (brunob) [reply preview/preview.js]: The key may be used more than that and if it's the case the demo page will show an error about that, maybe it's more safe to not show these tiles on the demo page.

Comment (yecarrillo) [reply preview/preview.js]: Done

**Comment by brunob** [comment]
GTM, @jieter any thoughts ?

**Comment by jieter** [comment]
Yes, looks good to me.

From experience with HERE, I suspect the key to generate more than 2k5 requests per day occasionally, so better to not have it in the preview.

**Pull request merged**